cmd/version.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+const version string = "0.0.1"
+
+var versionCmd = &cobra.Command{
```

Comment:
I don't think versionCmd is necessary.
Instead, you can set `version` field in rootCmd.